[2024-04-22T10:17:22.176+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-22T10:17:22.197+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-22T10:17:21.174510+00:00 [queued]>
[2024-04-22T10:17:22.203+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-22T10:17:21.174510+00:00 [queued]>
[2024-04-22T10:17:22.203+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-22T10:17:22.217+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): get_files> on 2024-04-22 10:17:21.174510+00:00
[2024-04-22T10:17:22.226+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'manual__2024-04-22T10:17:21.174510+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpqm7z1w8e']
[2024-04-22T10:17:22.228+0000] {standard_task_runner.py:91} INFO - Job 25: Subtask get_files
[2024-04-22T10:17:22.230+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=23268) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-22T10:17:22.230+0000] {standard_task_runner.py:63} INFO - Started process 23270 to run task
[2024-04-22T10:17:22.269+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.get_files manual__2024-04-22T10:17:21.174510+00:00 [running]> on host b6fcd665afc9
[2024-04-22T10:17:22.339+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-22T10:17:21.174510+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-22T10:17:21.174510+00:00'
[2024-04-22T10:17:22.340+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-22T10:17:22.341+0000] {logging_mixin.py:188} INFO - 5 files found in /opt/***/dags/data
[2024-04-22T10:17:22.342+0000] {python.py:237} INFO - Done. Returned value was: ['/opt/***/dags/data/github_events_01.json', '/opt/***/dags/data/github_events_03.json', '/opt/***/dags/data/github_events_02.json', '/opt/***/dags/data/github_events_05.json', '/opt/***/dags/data/github_events_04.json']
[2024-04-22T10:17:22.342+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-22T10:17:22.361+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, execution_date=20240422T101721, start_date=20240422T101722, end_date=20240422T101722
[2024-04-22T10:17:22.405+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-22T10:17:22.430+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-22T10:17:22.436+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
